### PR TITLE
fix(plugin-meetings): stop share not working when using meeting.stopS…

### PR DESCRIPTION
**Pull request by arun3528**
Fixes the meeting.stopShare which was being implemented wrong to just stop the share and not actually stopping the share 
Changed to use updateShare function

Fixes #[INSERT LINK TO ISSUE NUMBER]

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -3184,11 +3184,24 @@ export default class Meeting extends StatelessWebexPlugin {
...
+    });
+  }
+
+  /**
+   * sends stops floor request
+   * @returns {Promise} see #meetingRequest.changeMeetingFloor
+   * @private
+   * @memberof Meeting
+   */
+  _stopShare() {
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:3204]: Could we name this more descriptive?

```suggestion
  stopFloorRequest() {
```

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js [hunk 2]
```diff
@@ -1413,6 +1412,24 @@ describe('plugin-meetings', () => {
...
+      describe('#_stopShare', () => {
+        it('should have #_stopShare', () => {
+          assert.exists(meeting._stopShare);
+        });
+        beforeEach(() => {
+          meeting.locusInfo.mediaShares = [{name: 'content', url: url1}];
+          meeting.locusInfo.self = {url: url2};
+          meeting.meetingRequest.changeMeetingFloor = sinon.stub().returns(Promise.resolve());
+        });
+        it('should send the share', async () => {
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:1424]:
```suggestion
        it('should call change meeting floor', async () => {
```

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js [hunk 3]
```diff
@@ -896,16 +896,15 @@ describe('plugin-meetings', () => {
           assert.exists(meeting.stopShare);
         });
         beforeEach(() => {
-          meeting.locusInfo.mediaShares = [{name: 'content', url: url1}];
-          meeting.locusInfo.self = {url: url2};
-          meeting.meetingRequest.changeMeetingFloor = sinon.stub().returns(Promise.resolve());
+          meeting.mediaProperties.mediaDirection = {receiveShare: true};
+          meeting.updateShare = sinon.stub().returns(Promise.resolve());
         });
         it('should send the share', async () => {
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:902]: Let's update the description for what we are actually testing:

```suggestion
        it('should call updateShare', async () => {
```

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js]: Thanks @adamweeks  fixed the review comments

File: packages/node_modules/samples/browser-call-with-screenshare/app.js [hunk 4]
```diff
@@ -283,10 +283,7 @@ document.getElementById('stop-screen-share').addEventListener('click', () => {
   if (activeMeeting) {
     // First check if we can update, if not, wait and retry
     waitForMediaReady(activeMeeting).then(() => {
-      activeMeeting.updateShare({
-        sendShare: false,
-        receiveShare: true
-      });
+      activeMeeting.stopShare();
```
Comment (adamweeks) [inline packages/node_modules/samples/browser-call-with-screenshare/app.js:286]: Thanks for updating these!

**Review by adamweeks — APPROVED** [review]
LGTM!

**Pull request merged**